Can't open the app

### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App version

6.902 (39002)

### Where did you get the app from?

F-Droid

### Android version

9 (LineageOS unofficial)

### Device model

Lenovo Tab4 8 plus

### Steps to reproduce

Just opening the app crashes

### Expected behavior

Open normally

### Actual behavior

Crashes

### Logs

```
FATAL EXCEPTION: main
Process: com.fsck.k9, PID: 10578
java.lang.RuntimeException: Unable to resume activity {com.fsck.k9/com.fsck.k9.activity.MessageList}: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.widget.TextView.setText(java.lang.CharSequence)' on a null object reference
	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3822)
	at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3854)
	at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:51)
	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:145)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:70)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1816)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:193)
	at android.app.ActivityThread.main(ActivityThread.java:6718)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:491)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.widget.TextView.setText(java.lang.CharSequence)' on a null object reference
	at com.fsck.k9.activity.MessageList.setActionBarTitle(MessageList.kt:937)
	at com.fsck.k9.activity.MessageList.setMessageListTitle(MessageList.kt:951)
	at com.fsck.k9.ui.messagelist.MessageListFragment.setWindowTitle(MessageListFragment.kt:471)
	at com.fsck.k9.ui.messagelist.MessageListFragment.updateTitle(MessageListFragment.kt:434)
	at com.fsck.k9.ui.messagelist.MessageListFragment.onResume(MessageListFragment.kt:591)
	at androidx.fragment.app.Fragment.performResume(Fragment.java:3206)
	at androidx.fragment.app.FragmentStateManager.resume(FragmentStateManager.java:658)
	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:302)
	at androidx.fragment.app.FragmentStore.moveToExpectedState(FragmentStore.java:114)
	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1663)
	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:3247)
	at androidx.fragment.app.FragmentManager.dispatchResume(FragmentManager.java:3179)
	at androidx.fragment.app.FragmentController.dispatchResume(FragmentController.java:285)
	at androidx.fragment.app.FragmentActivity.onResumeFragments(FragmentActivity.java:333)
	at androidx.fragment.app.FragmentActivity.onPostResume(FragmentActivity.java:322)
	at androidx.appcompat.app.AppCompatActivity.onPostResume(AppCompatActivity.java:245)
	at android.app.Activity.performResume(Activity.java:7325)
	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3814)
	... 11 more
```

--- a/app/ui/legacy/src/main/res/layout/message_list.xml
+++ b/app/ui/legacy/src/main/res/layout/message_list.xml
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.drawerlayout.widget.DrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:id="@+id/drawerLayout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
@@ -12,48 +11,12 @@
         android:layout_height="match_parent"
         android:orientation="vertical">
 
-        <com.google.android.material.appbar.MaterialToolbar
+        <include
             android:id="@+id/toolbar"
-            style="?attr/toolbarStyle"
+            layout="@layout/message_list_toolbar"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:minHeight="?attr/actionBarSize"
-            android:layout_alignParentTop="true"
-            tools:navigationIcon="@drawable/ic_arrow_back">
-
-            <!-- We're not using MaterialToolbar's title/subtitle support because it is broken when using large system
-                 font sizes. See https://issuetracker.google.com/issues/135865267 -->
-            <LinearLayout
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:orientation="vertical"
-                android:layout_marginVertical="@dimen/toolbarTitleMarginVertical">
-
-                <com.google.android.material.textview.MaterialTextView
-                    android:id="@+id/toolbarTitle"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:singleLine="true"
-                    android:ellipsize="end"
-                    android:textAppearance="?attr/textAppearanceTitleLarge"
-                    android:textColor="?attr/colorOnSurface"
-                    tools:text="Inbox" />
-
-                <com.google.android.material.textview.MaterialTextView
-                    android:id="@+id/toolbarSubtitle"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:singleLine="true"
-                    android:ellipsize="end"
-                    android:textAppearance="?attr/textAppearanceTitleMedium"
-                    android:textColor="?attr/colorOnSurfaceVariant"
-                    android:visibility="gone"
-                    tools:visibility="visible"
-                    tools:text="demo@domain.example" />
-
-            </LinearLayout>
-
-        </com.google.android.material.appbar.MaterialToolbar>
+            android:layout_alignParentTop="true" />
 
         <ProgressBar
             android:id="@+id/message_list_progress"

--- a/app/ui/legacy/src/main/res/layout/message_list_toolbar.xml
+++ b/app/ui/legacy/src/main/res/layout/message_list_toolbar.xml
@@ -1,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.google.android.material.appbar.MaterialToolbar xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/toolbar"
+    style="?attr/toolbarStyle"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_alignParentTop="true"
+    android:minHeight="?attr/actionBarSize"
+    tools:navigationIcon="@drawable/ic_arrow_back">
+
+    <!-- We're not using MaterialToolbar's title/subtitle support because it is broken when using large system
+         font sizes. See https://issuetracker.google.com/issues/135865267 -->
+    <LinearLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginVertical="@dimen/toolbarTitleMarginVertical"
+        android:orientation="vertical">
+
+        <com.google.android.material.textview.MaterialTextView
+            android:id="@+id/toolbarTitle"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:ellipsize="end"
+            android:singleLine="true"
+            android:textAppearance="?attr/textAppearanceTitleLarge"
+            android:textColor="?attr/colorOnSurface"
+            tools:text="Inbox" />
+
+        <com.google.android.material.textview.MaterialTextView
+            android:id="@+id/toolbarSubtitle"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:ellipsize="end"
+            android:singleLine="true"
+            android:textAppearance="?attr/textAppearanceTitleMedium"
+            android:textColor="?attr/colorOnSurfaceVariant"
+            android:visibility="gone"
+            tools:text="demo@domain.example"
+            tools:visibility="visible" />
+
+    </LinearLayout>
+
+</com.google.android.material.appbar.MaterialToolbar>

--- a/app/ui/legacy/src/main/res/layout/split_message_list.xml
+++ b/app/ui/legacy/src/main/res/layout/split_message_list.xml
@@ -13,7 +13,7 @@
 
         <include
             android:id="@+id/toolbar"
-            layout="@layout/toolbar"
+            layout="@layout/message_list_toolbar"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:layout_alignParentTop="true" />

